projects/plugins/boost/app/assets/src/js/pages/index/index.tsx

```diff
@@ -127,12 +127,23 @@ const Index = () => {
 					</>
 				}
 				description={
-					<p>
-						{ __(
-							'Store and serve preloaded content to reduce load times and enhance your site performance and user experience.',
-							'jetpack-boost'
+					<>
+						<p>
+							{ __(
+								'Store and serve preloaded content to reduce load times and enhance your site performance and user experience.',
+								'jetpack-boost'
+							) }
+						</p>
+						{ site.isAtomic && (
+							<Notice
+								level="warning"
+								title={ __( 'Page Cache is unavailable', 'jetpack-boost' ) }
+								hideCloseButton={ true }
+							>
+								<p>{ __( 'Atomic sites already have caching enabled.', 'jetpack-boost' ) }</p>
```

Comment:
Looks great, but I would change the warning message:
```suggestion
								<p>{ __( 'Your website already has a page cache running on it powered by WordPress.com.', 'jetpack-boost' ) }</p>
```